include/engine/plugins/trip.hpp

```diff
@@ -145,7 +137,7 @@ template <class DataFacadeT> class RoundTripPlugin final : public BasePlugin
             // whole graph is one scc
             std::vector<NodeID> location_ids(number_of_locations);
             std::iota(std::begin(location_ids), std::end(location_ids), 0);
-            return SCC_Component(std::move(location_ids));
+            return SCC_Component(std::move(location_ids), {0, location_ids.size()});
```

Comment:
This is the same problem we had in the prototype. Moving in the second part breaks. The SCC_Component will see {0,0} instead of {0. location_ids.size()}.